lib/index.js

```diff
@@ -2,13 +2,73 @@
 
 // Load modules
 
-const Http = require('http');
 const Hoek = require('hoek');
 
 
 // Declare internals
 
-const internals = {};
+const internals = {
+    STATUS_CODES: {
+        __proto__: null,
```

Comment:
Security? And thanks for the info, was thinking of why not a map but read up on mdn and this approach does seem better for this use case.